lib/config/defaults.js

```diff
@@ -466,6 +468,14 @@ const applyModuleDefaults = (
 				...wasm
 			});
 		}
+		if (asset) {
+			rules.push({
+				test: /\.js$/,
```

Comment:
Doesn't make sense here. new URL should work for all files. This is not the origin, but the target.

Reply:
ah yes, thanks for pointing